Source/Core/Common/Network.cpp

```diff
@@ -18,6 +18,15 @@
 
 #include <fmt/format.h>
 
+#ifdef __linux__     // features.h is only available on Linux
+#define _GNU_SOURCE  // We have to define this to use macros in features.h
+#include <features.h>
+#ifndef __USE_GNU
+#define __MUSL__
```

Comment:
I don't know the details but doing this doesn't seem to follow what the authors of this lib has in mind: https://wiki.musl-libc.org/faq.html
> Why is there no __MUSL__ macro? (...)

Reply:
Ah, right. It's not quite a musl macro, it's more of a "not glibc" macro, at least in this context, since we're checking if we aren't using glibc. I suppose the naming of the macro isn't *quite* right.